client/python/samples/resnet_grpc_predict_binary.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import numpy as np
+import classes
+from ovmsclient import make_grpc_client, make_grpc_predict_request
+
+parser = argparse.ArgumentParser(description='Make prediction using images in binary format')
+parser.add_argument('--images_list', required=False, default='./utils/resnet_images.txt',
```

Comment:
Lets go with `--images_dir` and drop accuracy check like with non-binary version